src/mainwindow.cpp

```diff
@@ -197,7 +197,6 @@ void MainWindow::tableOfContentToggled(bool state)
     if (state) {
         checkActionNoSignal(KiwixApp::ToggleReadingListAction, false);
         mp_ui->sideBar->setCurrentWidget(mp_ui->tableofcontentbar);
-        mp_ui->sideBar->setStyleSheet("background-color: white;");
```

Comment:
This change shouldn't be there in the first place. Do not let things like this appear in the commit history.